lib/codebreaker/game.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Codebreaker
+  class Game
+    include Validations
+    include Constants::Shared
+
+    attr_reader :user, :secret_code, :attempts, :hints, :type, :secret_code_for_hints, :win
```

Comment:
Are u sure u need all these readers? secret_code - reader? It isn't secure. Think about it